llvm/lib/Target/X86/X86InstrInfo.cpp

```diff
@@ -3120,7 +3123,8 @@ bool X86InstrInfo::hasCommutePreference(MachineInstr &MI, bool &Commute) const {
 
 int X86::getCondSrcNoFromDesc(const MCInstrDesc &MCID) {
   unsigned Opcode = MCID.getOpcode();
```

Comment:
It seems the support for CFCMOV in X86FlagsCopyLowering.cpp  is missing.

Reply:
Also miss for cmovconversion

Reply:
Will do.